husky_control/package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <name>husky_control</name>
-  <version>0.6.4</version>
+  <version>0.6.5</version>
```

Comment:
Please do not edit the package version; this gets updated when the package is released

Reply:
Sorry, did not know that this gets updated automatically when the package is released. The requested changes have been made.